src/main/java/com/example/bankservicedemo/service/impl/UserServiceImpl.java

```diff
@@ -45,4 +45,14 @@ public void remove(Long id) {
     public List<User> findAll() {
         return userRepository.findAll();
     }
+
+    @Override
+    public User getById(Long userId) {
+        return userRepository.getOne(userId);
```

Comment:
will you be able to get roles of this user?

same question for your `getByPhoneNumber()` method
see checklist for HW-1 - Be sure to fetch roles along with the User.